src/util/config/config.h

```diff
@@ -120,6 +120,10 @@ namespace dxvk {
     static bool parseOptionValue(
       const std::string&  value,
             int32_t&      result);
+
+    static bool parseOptionValue(
+      const std::string&  value,
+            uint32_t&     result);
```

Comment:
Any particular reason why we need unsigned integers as well? Currently we're using `int32_t` for all the options, if the user enters a negative number it's their own fault.

Reply:
A uint32_t works out nicely here because the max value of GetAvailableTextureMem is UINT32_MAX (the counting is done from min(vramAmount, UINT32_MAX))

We can then limit the amount of VRAM exposed to the game/tracking via the option. 

https://github.com/Joshua-Ashton/d9vk/blob/master/src/d3d9/d3d9_options.cpp#L52

Reply:
Wouldn't it be more useful to specify this in Megabytes rather than bytes, for user convenience?

Reply:
Some Sims 2 people are using their own dxvk.confs right now and I am wary of breaking that and getting 100 of them on my tracker 🐸 

I'll make that change anyway though.